DirectEmployers/MyJobs
# A better fix for the ambigoustimerror

## What's this PR do?
According to SO this is a more reliable fix for the AmbiguousTimeError I was seeing during some imports. 
https://stackoverflow.com/questions/21465528/resolving-ambiguoustimeerror-from-djangos-make-aware

## How should this be manually tested?
Run a business unit experiencing this issue through imports about 50 times. Confirm it always works.
For some reason this is flakey. I don't know why. I imagine it would be much more prevalent/easy to reproduce around the DST changeover.
